wremnants/muon_calibration.py

```diff
@@ -31,9 +31,19 @@ def make_muon_calibration_helpers(mc_filename=data_dir+"/calibration/correctionR
 
     return mc_helper, data_helper, uncertainty_helper
 
-def make_muon_bias_helpers(lbl=False):
+def make_muon_bias_helpers(corr_type, smearing=True):
```

Comment:
I think the bias and the smearing are separate things, probably another name is more appropriate.

Reply:
Ah ok I see.